gulpfile.js

```diff
@@ -35,18 +36,19 @@ gulp.task('scss', function() {
     .pipe(gulp.dest('dist/css'));
 });
 
-gulp.task('webserver', function() {
+gulp.task('webserver',['default'], function() {
   connect.server({
     port: 8000,
+    root: 'dist',
     host: 'techweek.dev',
     livereload: true
   });
 });
 
 
 gulp.task('fonts', function() {
-   return gulp.src('./node_modules/materialize-css/fonts/**')
-       .pipe(gulp.dest('dist/fonts'));
+  return gulp.src('./node_modules/materialize-css/fonts/**')
```

Comment:
I should probably make a new issue for this, would moving to https://github.com/material-components/material-components-web be beneficial seeing as its the (yet another) official Material Design framework?